bin/create_db.sh

```diff
@@ -19,22 +19,66 @@ psql -c "CREATE DATABASE chronicle ENCODING 'UTF-8' LC_COLLATE = 'en_US.UTF-8' L
 psql -c 'GRANT ALL PRIVILEGES ON DATABASE chronicle to chronicle;' -U $PSQLUSER
 psql -c 'ALTER SCHEMA public OWNER TO chronicle;' -U $PSQLUSER
 psql -c "CREATE TABLE IF NOT EXISTS users (
-  fxaId CHAR(32) PRIMARY KEY,
+  user_id CHAR(32) PRIMARY KEY,
   email VARCHAR(255) NOT NULL,
-  oauthToken TEXT,
-  createdAt TIMESTAMPTZ(3) NOT NULL,
-  updatedAt TIMESTAMPTZ(3)
+  oauth_token TEXT,
+  created_at TIMESTAMPTZ(3) NOT NULL,
+  updated_at TIMESTAMPTZ(3)
 );" -d chronicle -U chronicle
-psql -c "CREATE TABLE IF NOT EXISTS visits (
+psql -c "CREATE TABLE IF NOT EXISTS user_pages (
   id UUID PRIMARY KEY,
-  fxaId CHAR(32) REFERENCES users,
-  url TEXT NOT NULL,
-  rawUrl TEXT NOT NULL,
-  urlHash CHAR(40) NOT NULL,
+  user_id CHAR(32) REFERENCES users(user_id),
+  url VARCHAR(2048) NOT NULL,
+  raw_url VARCHAR(2048) NOT NULL,
+  url_hash CHAR(40) NOT NULL,
   title TEXT NOT NULL,
-  visitedAt TIMESTAMPTZ(3) NOT NULL,
-  updatedAt TIMESTAMPTZ(3)
+  extracted_at TIMESTAMPTZ(3),
+  extracted_author_name TEXT,
+  extracted_author_url VARCHAR(2048),
+  extracted_cache_age BIGINT,
+  extracted_content TEXT,
+  extracted_description TEXT,
+  extracted_favicon_color TEXT,
+  extracted_favicon_url VARCHAR(2048),
+  extracted_image_caption TEXT,
+  extracted_image_color TEXT,
+  extracted_image_entropy DOUBLE PRECISION,
+  extracted_image_height INTEGER,
+  extracted_image_url VARCHAR(2048),
+  extracted_image_width INTEGER,
+  extracted_language TEXT,
+  extracted_lead TEXT,
+  extracted_media_duration INTEGER,
+  extracted_media_height INTEGER,
+  extracted_media_html TEXT,
+  extracted_media_type TEXT,
+  extracted_media_width INTEGER,
+  extracted_provider_display TEXT,
+  extracted_provider_name TEXT,
+  extracted_provider_url VARCHAR(2048),
+  extracted_published TIMESTAMPTZ(3),
+  extracted_safe BOOLEAN,
```

Comment:
When `safe` is false, we also get two other fields from Embed.ly which we may need to store: http://embed.ly/docs/api/extract/features/safe

Also (from same doc):
> Once a url is flagged as malicious, you must respect the `cache_age` attribute. When `cache_age` expires, you must not show the message again until you have reevaluated that url with Embedly. This helps to mitigate false positives and ensure compliance with the terms of Google’s Safe Browsing API.

... Which I guess is OK since we're storing the `cache_age` above. We may just need a bit more logic around whether we need to priority rescrape a page to see if it is still on the blacklist.